whiteboard.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.5'
+
+services:
+  whiteboard:
+        image: tgrl/excalidraw-backend:latest
+        restart: ${RESTART_POLICY:-unless-stopped}
+        depends_on:
+            - web
+        networks:
+            meet.jitsi:
+              aliases:
+                    - ${WHITEBOARD_URL_BASE:-whiteboard.meet.jitsi}
```

Comment:
Just "hardcode" the alias, like we do for Etherpad.

There is not much of a point in changing it since it's just the internal network alias.

Reply:
Is this still ongoing? Or is this now supported natively already?

Reply:
Still ongoing